util.go

```diff
@@ -51,3 +56,61 @@ func BashComplete(c *cli.Context) {
 		PrintFlagNames(c.App.VisibleFlags(), c.App.Writer)
 	}
 }
+
+func ConfigPath() (string, error) {
+	// default config file path in `/etc/rhc/config.toml`
+	filePath := filepath.Join("/etc", LongName, "config.toml")
+
+	if _, err := os.Stat(filePath); os.IsNotExist(err) {
+		return "", nil
+	}
+
+	return filePath, nil
+}
+
+// Get the API server URL based on, insights-client.conf and rhsm.conf
+// This URL may differ from prod, stage and Satellite
+func GuessAPIURL() (string, error) {
+	var uString string
+	var baseURL *url.URL
+	// First check if the server api is set in insights conf
+	cfg, err := ini.Load("/etc/insights-client/insights-client.conf")
```

Comment:
FWIW, I've written an [INI parsing package](https://git.sr.ht/~spc/go-ini/) that follows the Go standard `Marshal`/`Unmarshal` pattern. I never liked how this ini package doesn't follow that structure.

Reply:
It's really nice package, but it is not on GitHub and probably nobody knows about it.

Reply:
I do mirror it to Github: https://github.com/subpop/go-ini

Reply:
I have changed the ini-parser to the one created by @subpop. I prefer to use our tools than external modules as long as they are simpler. 
It is not as straightforward to use, I needed to create some structs to read the parameters, and also added to the struct some tags to read the hyphen parameter names.

Reply:
Yea, you do have to define the file you're parsing as structs. I don't think I added support for unmarshalling into a map. I think it (currently) has to be a struct with struct tags.